### ISSUE TITLE
Make Column swapper component

### ISSUE THREAD

**Issue by ymtszw** [ymtszw/zephyr#58]
So that we can easily test/debug drag&drop column swap in PatternLab!

- [ ] Pull `order` array from ColumnStore
- [ ] Contain it in a component with drag/swap-related messages 
- [ ] Prepare well-defined init/add/remove APIs with update

Then, save the component state in PatternLab's Model, initialize it from column list corresponding to `numColumns`, add on AddColumn message... SHOULD work.

**Comment by ymtszw** [comment]
Flip-move animation would be cool, but is somewhat tricky to implement.

~For flexbox, `order` property can be used and animated, though it confuses keyboard (source-order-based ) navigation and screen readers.
One idea is, 1) animate (transition) using `order` property briefly, then 2) on `transitionend` or `animationend`, actually reorder nodes on source.
This SHOULD result in keyboard-navigate-friendly source code, while animating flex items.
Not tested, though.~
`order` prop animates only in "integer", meaning, it animates, but in discrete manner.
So it can change their positions "step-by-step" or "slot-by-slot" within animation duration, but not smoothing in-between.

Basically, flip animation requires more fine-grained position calculation, I assume.

**Comment by ymtszw** [comment]
I think this won't make it.

1. Order-aware, and flippable data structure is rather complicated in itself, and too "logic-y"
   - And it is already constructed as `ColumnStore`
2. When things become logic-y, it deviates too much from the concept of "View Model", where you should grasp essential, and concise information for a view
    - It should think in "snapshot" manner, not too stateful
3. Complicated, order-aware Model will be eventually wired to simple and essential View Model at Pages.Main. So let us do it there.
